inverse_covariance/tests/rank_correlation_test.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+from sklearn.utils.testing import assert_array_almost_equal
+
+from inverse_covariance.rank_correlation import (
+    _compute_ranks,
+    spearman_correlation,
+)
+
+Y = np.ones(shape=[10, 2])
+X = .001 * np.random.randn(20, 1)
+X = np.append(X, X, axis=1)
+
+
+def test_compute_ranks():
+    Y1 = 5.5 * Y
+    Y2 = _compute_ranks(Y)
+    assert_array_almost_equal(Y1, Y2)
+
+
+def test_spearman_correlation():
```

Comment:
Yeah, I will add something to the monte carlo profiling hopefully on the correctness side. Unfortunately it looks like scipy.stats does not have very statistically good estimators for rank related stuff. I will add in some classic examples from R to compare, but not super hopeful.